projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -140,7 +140,12 @@ const SecondaryButton = props => {
 		return false;
 	}
 
-	return <Button { ...props }>{ label }</Button>;
+	// Button component doesn't take these props
+	const buttonProps = { ...props };
+	delete buttonProps.shouldShowButton;
+	delete buttonProps.positionFirst;
```

Comment:
That might be another way to do that, maybe better as we don't mutate the object state?
```suggestion
	const { shouldShowButton, positionFirst, ...buttonProps } = props;
```

Reply:
Nice improvement, I did not think of this way when doing it. Technically I wasn't mutating the object state, it was creating a copy of it, but I still like your way better